packages/tasks/src/pipelines.ts

```diff
@@ -248,6 +237,14 @@ export const PIPELINE_DATA = {
 				type: "dialogue-modeling",
 				name: "Dialogue Modeling",
 			},
+			{
+				type: "dialogue-generation",
+				name: "Dialogue Generation",
+			},
+			{
+				type: "conversational",
+				name: "Conversational",
+			},
```

Comment:
still added it as a subtask under `text-generation` @osanseviero, i think it makes sense no?

Reply:
Yes, let's keep `conversational` as a subtask! As a follow-up at one point, it would be great to make conversational models more discoverable